catalog/harvest/catalog-harvest-webdav/src/main/java/org/codice/ddf/catalog/harvest/webdav/WebDavHarvester.java

```diff
@@ -241,15 +244,22 @@ public void setAttributeOverrides(List<String> incomingAttrOverrides) {
             String.format("Invalid attribute override key value pair of [%s].", keyValuePair));
       }
 
-      attributeOverrides.put(parts[0], parts[1]);
+      final String newValue = parts[1];
+      attributeOverrides.merge(
+          parts[0],
+          Arrays.asList(newValue),
+          (oldValue, v) -> {
+            oldValue.add(newValue);
+            return oldValue;
+          });
     }
   }
 
-  public void setWebdavAddress(String webdavAddress) {
+  private void setWebdavAddress(String webdavAddress) {
```

Comment:
I think since these are setters they should be public for convention.

Reply:
The public setter is `updateCallback`. `updateCallback` is the only setter that should be used because it does some argument validation and also reinitializes the Harvester.